anthos-bm-gcp-terraform/modules/vm/main.tf

```diff
@@ -21,6 +21,7 @@ module "external_ip_addresses" {
 
 module "compute_instance" {
   source            = "terraform-google-modules/vm/google//modules/compute_instance"
+  version           = "~> 6.3.0"
```

Comment:
Is the idea that this was the version used and worked until `6.4.0` came out yesterday[1] with a change that broke compatibility, and as thus we want to pin it to the last known working version?
1: https://github.com/terraform-google-modules/terraform-google-vm/releases

Reply:
@bourgeoisor - Yes that correct. With the new version there are various changed to be made to the scripts. SO pinning to the minor version for now.